modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/build.gradle

```diff
@@ -1,4 +1,5 @@
-#if (${projectType} != "workspace")buildscript {
+#if (${projectType} != "workspace")
+buildscript {
```

Comment:
This needs to be on the line above, because of how velocity handles whitespace.  :(

Reply:
Ok. Fixed. Thanks.